torch/nn/functional.py

```diff
@@ -2815,6 +2815,25 @@ def multi_margin_loss(input, target, p=1, margin=1., weight=None, size_average=N
     torch.Size([1, 1, 12, 12])
 """)
 
+pixel_unshuffle = _add_docstr(torch.pixel_unshuffle, r"""
+pixel_unshuffle(input, downscale_factor) -> Tensor
+
+Reverses the :class:`~torch.nn.PixelShuffle` operation by rearranging elements in a tensor of shape :math:`(*, C, H \times r, W \times r)` to a tensor of shape :math:`(*, C \times r^2, H, W)`, where r is a downscale factor.
```

Comment:
This might cause lint problem. 
max number of character per line is 120.

Reply:
Thanks! Just setup my local dev environment and the python linter (flake8) is not functioning correctly yet :(